crypto/x509/t_x509.c

```diff
@@ -94,11 +94,11 @@ int X509_print_ex(BIO *bp, X509 *x, unsigned long nmflags,
         }
         if (l != -1) {
             if (bs->type == V_ASN1_NEG_INTEGER) {
-                l = -l;
+                l = -(unsigned long)l;
```

Comment:
Once again, this is purely theoretical, as we don't actually support non-two's-complements, and formally we don't *have to* discuss this, so feel free to disregard...